go.sum

```diff
@@ -1,23 +1,137 @@
 cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
+github.com/BurntSushi/toml v0.3.1 h1:WXkYYl6Yr3qBf1K79EBnL4mak0OimBfB0XUf9Vl28OQ=
```

Comment:
I'm also not totally sure how go modules work, but I don't think you need all of the additional changes to this file. What happens if you run `go mod tidy` on this branch?